generate/config.go

```diff
@@ -36,6 +37,10 @@ type Config struct {
 	// The directory of the config-file (relative to which all the other paths
 	// are resolved).  Set by ValidateAndFillDefaults.
 	baseDir string
+
+	// The version of genqlient, from runtime.ReadBuildInfo, included in the
+	// generated files.  Set by ValidateAndFillDefaults.
+	Version string
```

Comment:
You can initialize this to a default value of `(default)` and still it be overridden at build time by the linker flags. 

I often do:
```
// 	Populated during build
var (
	AppName      = "unknown" // What the app is called
	Project      = "unknown" // Which GCP Project e.g. khan-internal-services
	Date         = ""        // Build Date in RFC3339 e.g. $(date -u +"%Y-%m-%dT%H:%M:%SZ")
	GitCommit    = "?"       // Git Commit sha1 of source
	Version      = "v0.0.0"  // go mod version: v0.0.0-20200214070026-92e9ce6ff79f
	HumanVersion = fmt.Sprintf(
		"%s %s %s (%s) on %s",
		AppName,
		Project,
		Version,
		GitCommit,
		Date,
	)
)
```

Reply:
To my mind, the correct Version is whatever would end up in go.mod if you did `go get genqlient@<head>`.  I don't know how easy that is to calculate though.

Reply:
Inside genqlient it is:
```
go list -f '{{.Version}}' -m github.com/Khan/genqlient@$(git rev-parse HEAD)
```
Outside genqlient in an application that has genqlient in it's go.mod, it is:
```
go list -m -f '{{ .Version }}' github.com/Khan/genqlient)" 
```